### PR TITLE
(DO NOT MERGE) Merge CaSSIS changes into dev.

### PR DESCRIPTION
**Pull request by krlberry**

### CODE REVIEW

File: isis/src/control/apps/cnetwinnow/cnetwinnow.cpp [hunk 1]
```diff
@@ -322,6 +322,23 @@ void IsisMain() {
     if (net.GetSerialConnections().size() > numInitialIslands) islandFlag = false; //test failed
     else islandFlag = true; //test passed
 
+    //Check to see if the network has split
```
Comment (celbal) [inline isis/src/control/apps/cnetwinnow/cnetwinnow.cpp:325]: Can we remove this commented out code?

File: isis/src/control/objs/ControlCubeGraphNode/ControlCubeGraphNode.cpp [hunk 2]
```diff
@@ -0,0 +1,269 @@
+#include "IsisDebug.h"
```
Comment (celbal) [inline isis/src/control/objs/ControlCubeGraphNode/ControlCubeGraphNode.cpp:1]: Wasn't this object deleted?

File: isis/src/control/objs/ControlCubeGraphNode/ControlCubeGraphNode.h [hunk 3]
```diff
@@ -0,0 +1,118 @@
+#ifndef ControlCubeGraphNode_h
```
Comment (celbal) [inline isis/src/control/objs/ControlCubeGraphNode/ControlCubeGraphNode.h:1]: Wasn't this deleted?

File: isis/src/control/objs/ControlMeasure/ControlMeasure.cpp [hunk 4]
```diff
@@ -30,6 +30,7 @@
 #include "ControlMeasureLogData.h"
```
Comment (celbal) [inline isis/src/control/objs/ControlMeasure/ControlMeasure.cpp:30]: ControlCubeGraphNode was deleted

File: isis/src/control/objs/ControlMeasure/ControlMeasure.h [hunk 5]
```diff
@@ -36,6 +36,7 @@ namespace Isis {
   class Camera;
   class ControlMeasureLogData;
   class ControlPoint;
+  class ControlCubeGraphNode;
```
Comment (celbal) [inline isis/src/control/objs/ControlMeasure/ControlMeasure.h:39]: ControlCubeGraphNode was deleted

File: isis/src/control/objs/ControlNet/ControlNet.h [hunk 6]
```diff
@@ -20,7 +20,7 @@
  *   $ISISROOT/doc//documents/Disclaimers/Disclaimers.html
  *   in a browser or see the Privacy &amp; Disclaimers page on the Isis website,
  *   http://isis.astrogeology.usgs.gov, and the USGS privacy and disclaimers on
- *   http://www.usgs.gov/privacy.html
+ *   http://www.usgs.gov/privacy.html.
```
Comment (celbal) [inline isis/src/control/objs/ControlNet/ControlNet.h:23]: I think you need to merge in the IPCE changes

File: isis/src/hayabusa/apps/amica2isis/amica2isis.cpp [hunk 7]
```diff
@@ -103,25 +103,30 @@ void IsisMain ()
   // Create a PVL to store the translated labels in
```
Comment (celbal) [inline isis/src/hayabusa/apps/amica2isis/amica2isis.cpp:103]: You need a history comment in amica2isis.xml

File: isis/src/hayabusa/apps/hyb1pds4gen/hyb1pds4gen.cpp [hunk 8]
```diff
@@ -0,0 +1,163 @@
+#include "Isis.h"
```
Comment (celbal) [inline isis/src/hayabusa/apps/hyb1pds4gen/hyb1pds4gen.cpp:1]: Does the naming convention of this app follow the ISIS standards for naming conventions?

File: isis/src/qisis/apps/ipce/IpceMainWindow.h [hunk 9]
```diff
@@ -24,7 +24,6 @@
  */
```
Comment (celbal) [inline isis/src/qisis/apps/ipce/IpceMainWindow.h:24]: I think you need to merge in the IPCE changes

File: isis/src/qisis/objs/CnetEditorWidget/APrioriLatitudeFilter.cpp [hunk 10]
```diff
@@ -2,43 +2,40 @@
 
 #include "APrioriLatitudeFilter.h"
```
Comment (celbal) [inline isis/src/qisis/objs/CnetEditorWidget/APrioriLatitudeFilter.cpp:3]: Should this be Apriori rather than APriori?

**Review by celbal — CHANGES_REQUESTED** [review]
So git is not correctly reflecting your changes. It's thinking that all of the IPCE changes are conflicts and that we changed a lot of files that we didn't.

I suggest that you make a new branch and copy the files that were changed rather than trying to deal with all of these problems.

**Comment by krlberry** [comment]
@makaylas :+1: Thank you, you're right about this missing the ipce changes.

**Comment by krlberry** [comment]
See Jeannie's PR #399

**Pull request closed without merging**